tests/SelfTest/UsageTests/Benchmark.tests.cpp

```diff
@@ -97,7 +97,7 @@ TEST_CASE("Benchmark containers", "[!benchmark]") {
         };
         int sum = 0;
         for (int i = 0; i < size; ++i)
-            sum += array[i];
+            sum += array[i]; // NOLINT(clang-analyzer-core.uninitialized.Assign)
```

Comment:
Wouldn't a better fix be changing `int array[size]` into `int array[size]{}` above? The test below would still require the change in `BENCHMARK` to happen to pass, but it should shut up the analyzer.

Reply:
Done.